src/registryDAO/types.mligo

```diff
@@ -16,8 +16,9 @@ type update_receiver_param =
   | Add_receivers of (address list)
   | Remove_receivers of (address list)
 
-type normal_proposal =
+type transfer_proposal =
```

Comment:
I considered this, in fact at first I didn't name the new type `transfer_proposal`.

However the problem isn't in the name, but in the fact that the older  and common `transfer_proposal` contains the `agora_post_id`:
```ocaml
type transfer_proposal =  { agora_post_id : nat  ; transfers : transfer_type list  }
```
which is fine for treasuryDAO, but not for registry, because we want a single `agora_post_id` for the whole neew type:
```ocaml
type transfer_proposal = 
  { agora_post_id : nat
  ; transfers : transfer_type list
  ; registry_diff : registry_diff  }
```
which means that the only part in common between the two is now `transfer_type list` and it isn't much useful to make this a common type :shrug: